docker-compose.yml

```diff
@@ -47,4 +47,13 @@ services:
       - $PWD/s3-mock/data:/data
     environment:
       MINIO_ACCESS_KEY: ${MINIO_ACCESS_KEY}
-      MINIO_SECRET_KEY: ${MINIO_SECRET_KEY}
\ No newline at end of file
+      MINIO_SECRET_KEY: ${MINIO_SECRET_KEY}
+
+  # use nginx to serve static pages    
```

Comment:
I think this comment might be more useful to be something along the lines of "Mimick the public facing side of an S3 bucket to serve static pages"